docs/modules/ROOT/pages/gsn.adoc

```diff
@@ -0,0 +1,77 @@
+= The Gas Station Network
+
+User onboarding is one of the hottest topics in Ethereum, and meta-transactions the de-facto solution to this important problem. Here you'll learn all about the Gas Station Network (GSN) and how to write contracts that don't require their users to hold Ether to pay for gas.
+
+If you're already up to speed with the workings of the GSN, feel free to skip to [receiving-a-relayed-call], where we'll go over how to use the OpenZeppelin Contracts to easily write a GSNRecipient contract. Otherwise, strap in!
+
+== Sending gas-less transactions
+
+All Ethereum transactions spend gas, and the sender of the transaction must have enough Ether to pay for these gas costs. Even though these costs are low for basic transactions (a couple cents), getting this Ether is no easy task: you'll probably need to go through Know Your Customer and Anti Money-Laundering processes (KYC & AML), which not only take time but often involve sending a selfie holding your passport over the Internet (!). Only the most hardcore users will put up with this hassle, and dApp adoption greatly suffers when Ether is required. We can do better.
+
+**Enter meta-transactions**. This is a fancy name for a simple idea: a third-party can send another user's transactions and pay themselves for the gas cost. That's it! There's some tricky technical details, but those can be safely ignored when interacting with the GSN. This way, instead of your users calling into your contract (called the _recipient_) directly, someone else (we'll call them a _relay_) will send their transaction and pay for the cost.
+
+But why would they do such a thing?
+
+== Incentives
+
+Relays are not running a charity: they're running a bussiness. The reason why they'll gladly pay for your users' gas costs is because they will in turn charge your contract, the recipient. That way relays get their money back, plus a bit extra as a _fee_ for their services.
+
+This may sound strange at first, but paying for user onboarding is a very common bussiness practice. Lots of money are spent on advertising, free trials, new user discounts, etc., all with the goal of user acquisition. And compared to those, the cost of a couple Ethereum transactions is actually very small.
```

Comment:
If you want to throw in the technical term, you can refer to Customer Acquisition Cost.

Reply:
I'd rather not go all buzzwordy, though a link to e.g. wikipedia might be useful to give people some more context.